workspaces/entity-feedback/plugins/entity-feedback-backend/src/service/router.ts

```diff
@@ -227,9 +227,16 @@ export async function createRouter(
         type: 'entity',
         entityRef: entityOwner,
       };
+
+      // Construct entity URL from entityRef (format: "kind:namespace/name")
+      const [kind, namespaceName] = req.params.entityRef.split(':');
+      const [namespace, name] = namespaceName.split('/');
+      const entityUrl = `/catalog/${namespace}/${kind}/${name}/feedback`;
```

Comment:
Thanks for suggesting this change however it is not guaranteed that this is always the entity URL - consumers are free to attach the catalog plugin and "feedback" tab to any custom route name of their choosing even though this is the default. Therefore we probably need a more configurable way to construct this url that is consistent with how the frontend plugin is setup